logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -1299,7 +1311,8 @@ class UserSessionScope internal constructor(
         get() = FeatureConfigEventReceiverImpl(
             GuestRoomConfigHandler(userConfigRepository, kaliumConfigs),
             FileSharingConfigHandler(userConfigRepository),
-            MLSConfigHandler(userConfigRepository, userId),
+            MLSConfigHandler(userConfigRepository, updateSupportedProtocolsAndResolveOneOnOnes, userId),
```

Comment:
here we can re-use those variable for example.

Reply:
Done